src/action_types/teams.js

```diff
@@ -37,6 +37,7 @@ export default keyMirror({
     SELECT_TEAM: null,
     UPDATED_TEAM: null,
     PATCHED_TEAM: null,
+    REGENERATE_TEAM_INVITE_ID: null,
```

Comment:
Minor nit since this isn't actually used anywhere, but I think this should be `REGENERATED_TEAM_INVITE_ID`. We tend to use the past tense for things that happen in response to an API call
```suggestion
    REGENERATED_TEAM_INVITE_ID: null,
```

Reply:
Applied, thank you!